embark/templates/tracker/device.html

```diff
@@ -0,0 +1,60 @@
+{% extends "base.html" %}
+
+<!--
+    Input: device id/name/label
+
+        Label(X-axis): FW Version /Analysis or notes
+        Data(y-axis):  basically all models.Results
+            Can be Sortable by:    CVE count
+            Kernel version
+            ...etc
+
+        Canvas 1: radio chart of all analysis
+        Canvas 2: Time graph over version sorted by creation date
+-->
+
+{% load static %}
+{% load django_bootstrap5 %}
+
+{% block style %}<link rel="stylesheet" type="text/css" href="{% static 'content/css/tracker.css' %}"/>{% endblock style %}
+{% block js %}
+    <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
```

Comment:
We need to note this somewhere that EMBArk is currently not running in offline environments. Probably open an issue for this

Reply:
As we recommend to use EMBA and EMBArk in a trusted environment we should make these scripts available offline.